openedx/core/djangoapps/schedules/tests/test_resolvers.py

```diff
@@ -99,7 +99,7 @@ def test_get_course_org_filter_exclude__in(self, course_org_filter, expected_org
             "Can't test schedules if the app isn't installed")
 @override_waffle_flag(COURSE_UPDATE_WAFFLE_FLAG, True)
 @freeze_time('2017-08-01 01:00:00', tz_offset=0, tick=False)
-class TestCourseUpdateResolver(SchedulesResolverTestMixin, ModuleStoreTestCase):
+class TestCourseUpdateResolver(SchedulesResolverTestMixin, CacheIsolationTestCase, ModuleStoreTestCase):
```

Comment:
I think you want `CacheIsolationMixin` instead, since `ModuleStoreTestCase` already inherists `TestCase`.

Reply:
@jmbowman I tried that, but it needed the setup/teardown methods from [`CacheIsolationTestCase`](https://github.com/edx/edx-platform/blob/829227ade3af9ffcb00195c09632442b48d46203/openedx/core/djangolib/testing/utils.py#L127) -- perhaps these should live in the mixin instead?